### ISSUE TITLE
Cache template poster images

### ISSUE THREAD

**Issue by BrittanyIRL** [swissspidy/web-stories-wp#9678]
Let's see about caching the template poster images so that after the initial load of the templates in the dashboard things are even faster! 

This update should improve the explore templates view as well as detail template view. 

You can see this by running lighthouse perf on the dashboard before and after making the update. Remember that locally running lighthouse is not the same as running it on staging (our dev environment is wayyyyy slower than prod).

**Comment by BrittanyIRL** [comment]
So i was a little confused by why this was coming up in lighthouse given we _are_ caching things. 

(network in staging sampling)
<img width="592" alt="Screen Shot 2021-12-02 at 3 15 25 PM" src="https://user-images.githubusercontent.com/10720454/144511897-d0b1addc-2076-4273-9a96-3059ce8e177c.png">

This is the diagnostic i was running off of: 
<img width="761" alt="Screen Shot 2021-12-02 at 3 13 46 PM" src="https://user-images.githubusercontent.com/10720454/144511989-eeda3ff1-d9f3-4c80-bd3d-dd0fe9a619f5.png">

So i thought there was some sort of discrepancy.  I forgot that lighthouse is clearing storage - so it wasn't allowed to reference any cache at all. If I uncheck the 'clear storage' button then the cache sticks around and all of this is working just fine. I wanted to double check because I was worried about why it wasn't matching up. 

TL;DR: we're fine - checking settings is important and not a good thing to assume.

<img width="806" alt="Screen Shot 2021-12-02 at 3 14 35 PM" src="https://user-images.githubusercontent.com/10720454/144512187-fab2b400-f7e5-49e4-bb46-45ecb9c33c6d.png">
